packages/react-components/src/components/Switch/Switch.spec.tsx

```diff
@@ -62,4 +62,12 @@ describe('Switch', () => {
     expect(checkbox.disabled).toEqual(true);
     expect(lockIcon).toBeVisible();
   });
+
+  it('should allow setting accessible name for input element', () => {
+    const label = 'Hello world';
+    const { getByRole } = render(<Switch on={true} ariaLabel={label} />);
+
+    const checkbox = getByRole('checkbox', { name: label });
```

Comment:
Or 
```suggestion
    const checkbox = getByRole('checkbox');
```

? 🤔

Reply:
`const checkbox = getByRole('checkbox');` should be enough, but I think it doesn't matter that much in this particular test case.

Reply:
I will change it, as with `const checkbox = getByRole('checkbox');` we should get better error if the test fails